packages/nextjs/app/builders/0x97d45F88468289ac299B79cF01bbB6f218c46230/page.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
```

Comment:
You don't need "use client" here, since this can be server-rendered (you are not doing any client actions: useEffect, etc..)